core/templates/core/panel.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Panel Gwardii{% endblock %}
+
+{% load list_events %}
+
+{% block content %}
+    <div class="absolute flex items-center md:hidden h-fit">
+        <button id="mobile-menu-btn" type="button" class="relative inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700">
+            <i class="fa-solid fa-bars block h-6 w-6"></i>
+        </button>
+    </div>
```

Comment:
This button can't look like this when clicked, make this background smaller or something
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/8d186e71-a2d7-4037-ae1b-d7a6e2542525)
Also leave more space between it and link to github
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/14ce2f7b-ead7-481f-9461-4117537a3630)